lib/logux/error_handler.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Logux
+  class ErrorHandler
+    attr_reader :exception
+
+    def initialize(exception)
+      @exception = exception
+    end
+
+    def message
+      case exception
+      when Logux::NoActionError
+        ['unknownAction', exception.meta.id]
+      when Logux::NoChannelError
+        ['unknownChannel', exception.meta.id]
+      when Logux::NoPolicyError
+        ['unknownPolicy', exception.meta.id]
+      else
+        [:error].to_json
```

Comment:
i think it would be better to keep the answer format unified for every error we have and render the same schema's json:
```ruby
[exeption.class.camelize(:lower), exception.meta.id]
```

Reply:
@ai I have changed a little format of error messages to unify format. Is it good for you?

Reply:
It is OK, but there is no such think as `unknownActionError`. It is OK to not have action handler. Only policy is required. This is why we need only `unknownAction` error (remove current `unknownAction` and rename `unkownPolice` to `unknownAction`).

Also we don’t need `Error` suffix. Unknown means always an error. Let’s make them shorter.

Reply:
I am a little confused. So as I understand correct:
- return `unknownAction` if `verify_authorized=true` and there are *NO POLICY*
- just literally do nothing if `verify_authorized=true` and there *POLICY* but *ACTION* is missing
- do nothing if `verify_authorized=false` and there are no *ACTION*
- drop the `Error` suffix

Reply:
Yeap :+1: